docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -169,107 +198,127 @@ quarkus.oidc.credentials.secret=secret
+. Open a link:{url-quarkusio-guides}dev-ui[Dev UI], which you can find at http://localhost:8080/q/dev-v1[/q/dev-v1], then click a `Provider: Keycloak` link in an `OpenID Connect` `Dev UI` card.
+. When prompted to log in to a `Single Page Application` provided by `OpenID Connect Dev UI`, do the following steps:
 
- * Login as `alice` (password: `alice`) who has a `user` role
- ** accessing `/api/admin` will return `403`
- ** accessing `/api/users/me` will return `200`
- * Logout and login as `admin` (password: `admin`) who has both `admin` and `user` roles
- ** accessing `/api/admin` will return `200`
- ** accessing `/api/users/me` will return `200`
+ * Log in as `alice` (password: `alice`), who has a `user` role.
+ ** Accessing `/api/admin` returns `403`.
+ ** Accessing `/api/users/me` returns `200`.
+ * Log out and log in as `admin` (password: `admin`), who has both `admin` and `user` roles.
+ ** Accessing `/api/admin` returns `200`.
+ ** Accessing `/api/users/me` returns `200`.
 
 === Run the Application in JVM mode
 
-When you're done playing with the `dev` mode" you can run it as a standard Java application.
-
-First compile it:
+When you are done with developer mode, you can run the application as a standard Java application.
 
+. Compile the application:
++
+====
 include::{includes}/devtools/build.adoc[]
-
-Then run it:
-
+====
+. Run the application:
++
+====
 [source,bash]
 ----
 java -jar target/quarkus-app/quarkus-run.jar
 ----
+====
 
-=== Run the application in Native mode
+=== Run the application in native mode
 
-This same demo can be compiled into native code: no modifications required.
+You can compile this same demo as-is into native mode without needing any modifications.
+This implies that you no longer need to install a JVM on your production environment.
+The runtime technology is included in the produced binary and optimized to run with minimal resources required.
 
-This implies that you no longer need to install a JVM on your production environment, as the runtime technology is included in the produced binary, and optimized to run with minimal resource overhead.
+Compilation takes a bit longer, so this step is disabled by default.
 
-Compilation will take a bit longer, so this step is disabled by default;
-let's build again by enabling the `native` profile:
+. Build your application again by enabling the `native` profile:
++
+====
 
 include::{includes}/devtools/build-native.adoc[]
-
-After getting a cup of coffee, you'll be able to run this binary directly:
-
+====
+. After waiting a little while, you run the following binary directly:
++
+====
 [source,bash]
 ----
 ./target/security-openid-connect-quickstart-1.0.0-SNAPSHOT-runner
 ----
+====
 
 === Test the application
 
-See the <<keycloak-dev-mode,Running the Application in Dev mode>> section above about testing your application in a dev mode.
+For information about testing your application in developer mode, see the preceding <<keycloak-dev-mode>> section.
```

Comment:
```suggestion
For information about testing your application in devmode, see the preceding <<keycloak-dev-mode>> section.
```

Reply:
I think `devmode` is a real word in the Quarkus vocabulary, @gastaldi @geoand is it the case ? I'm not sure `developer mode` works

Reply:
I see us using `development mode` in other docs so it's probably best to be consistent

Reply:
+1 to `Development mode`

Reply:
@sberyozkin , @gastaldi, @geoand - Thanks for flagging this and your comments.

As 'dev mode' is a proper noun defined by Quarkus and commonly used in this specialization,  I have now reverted back to `dev mode` for all instances except for the first one. 

My :tropical_fish: brain forgot about a similar discussion with Quarkus SMEs that came up earlier this year:

We concluded that to prevent ambiguity around the word `dev` (is it developer/development, it would be best practice to use the full form in the first instance on the page, for example, `development (dev) mode`, and use `dev mode` thereafter.

I will also clean up of other inconsistent instances across the other docs later **after 3.2 product GA** so that we don't create backport headaches. 

I updated the PR according to ^^. Please do shout if you disagree with this approach. Thanks again :green_heart:

Reply:
Thanks @michelle-purcell @geoand @gastaldi